examples/index.html

```diff
@@ -25,14 +25,24 @@ <h1>KVS WebRTC Test Page</h1>
     <div class="row loader"></div>
     <div id="main" class="d-none">
         <form id="form" onsubmit="return false">
-            <h4>KVS Endpoint</h4>
+            <h4 id="subheader">KVS Endpoint</h4>
             <div class="form-group has-validation" style="position: relative;">
                 <label for="region">Region</label>
                 <input type="text" class="form-control valid" id="region" placeholder="Region" value="us-west-2" autocomplete="off" required>
                 <datalist id="regionList"></datalist>
                 <div id="region-invalid-feedback" class="invalid-feedback"></div>
             </div>
-            <h4>AWS Credentials</h4>
+            <div class="form-check-inline">
```

Comment:
Wondering if we can move this to 'advanced' given that I don't think many users will need to touch or care about this setting?

Reply:
I thought about this, but since we are keeping the default to be legacy while the internet as a whole is all moving towards IPv6, we should maybe let it be better advertised that there is the dual-stack option, and easier to find.

Reply:
Do you think it would be more intuitive design maybe to consider using a toggle button group for the setting? 

IPV4 / Dual-Stack

Reply:
That does sound nice, yes let me try that out.